examples/webxr_xr_dragging_custom_depth.html

```diff
@@ -0,0 +1,459 @@
+
+				controller1.add( line.clone() );
+				controller2.add( line.clone() );
+
+				raycaster = new THREE.Raycaster();
+
+				//
+
+				window.addEventListener( 'resize', onWindowResize );
+
+			}
+
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+			}
+
+			function onSelectStart( event ) {
+
+				const controller = event.target;
+
+				const intersections = getIntersections( controller );
+
+				if ( intersections.length > 0 ) {
+
+					const intersection = intersections[ 0 ];
+
+					const object = intersection.object;
+					object.material.uniforms.emissive.value = 1;
+					controller.attach( object );
+
+					controller.userData.selected = object;
+
+				}
+
+				controller.userData.targetRayMode = event.data.targetRayMode;
+
+			}
+
+			function onSelectEnd( event ) {
+
+				const controller = event.target;
+
+				if ( controller.userData.selected !== undefined ) {
+
+					const object = controller.userData.selected;
+					object.material.uniforms.emissive.value = 0;
+					group.attach( object );
+
+					controller.userData.selected = undefined;
+
+				}
+
+			}
+
+			function getIntersections( controller ) {
+
+				controller.updateMatrixWorld();
+
+				tempMatrix.identity().extractRotation( controller.matrixWorld );
+
+				raycaster.ray.origin.setFromMatrixPosition( controller.matrixWorld );
+				raycaster.ray.direction.set( 0, 0, - 1 ).applyMatrix4( tempMatrix );
+
+				return raycaster.intersectObjects( group.children, false );
+
+			}
+
+			function intersectObjects( controller ) {
+
+				// Do not highlight in mobile-ar
+
+				if ( controller.userData.targetRayMode === 'screen' ) return;
+
+				// Do not highlight when already selected
+
+				if ( controller.userData.selected !== undefined ) return;
+
+				const line = controller.getObjectByName( 'line' );
+				const intersections = getIntersections( controller );
+
+				if ( intersections.length > 0 ) {
+
+					const intersection = intersections[ 0 ];
+
+					const object = intersection.object;
+					object.material.uniforms.emissive.value = 1;
+					intersected.push( object );
+
+					line.scale.z = intersection.distance;
+
+				} else {
+
+					line.scale.z = 5;
+
+				}
+
+			}
+
+			function cleanIntersected() {
+
+				while ( intersected.length ) {
+
+					const object = intersected.pop();
+					object.material.uniforms.emissive.value = 0;
+
+				}
+
+			}
+
+			//
+
+			function animate() {
+
+				renderer.setAnimationLoop( render );
+
+			}
+
+			function render() {
+
+				if (renderer.xr.hasDepthSensing() && ! isDepthSupplied) {
+
+					group.children.forEach(child => {
+
+						child.material.uniforms.depthColor.value = renderer.xr.getDepthTexture();
+						child.material.uniforms.depthWidth.value = 1680;
+						child.material.uniforms.depthHeight.value = 1760;
+						child.material.uniforms.NeedUpdate = true;
```

Comment:
I was search `NeedUpdate` keyword in `src`, the result show only show something about `material.uniformsNeedUpdate = false;`

`uniforms.NeedUpdate` should be `uniformsNeedUpdate`?

or maybe I miss something.

Reply:
Good catch! `child.material.uniforms.NeedUpdate` can just be removed. Would you like to do that with a PR?